mozilla/bedrock
# Add first-party data preferences page (Fixes #11533)

## One-line summary

This PR adds a data preferences page under `/privacy/websites/` that will allow visitors to opt-out of 1st-party Glean analytics.

## Significant changes and points to review

The page copy is currently undergoing legal review for approval. The main things to review are the functionality outlined below. For now this page won't do anything other than set a cookie, which will evantually be used in https://github.com/mozilla/bedrock/pull/11402. This page will also be linked to from our website privacy notice which will be added via a PR to the legal docs repo.

## Issue / Bugzilla link

#11533

## Checklist

If relevant:


## Testing

http://localhost:8000/en-US/privacy/websites/data-preferences/

To test this work:
* Clicking "Opt-out of first-party data collection" should set a cookie with an id of `moz-1st-party-data-opt-out` and a 1 year expiry.
* Clicking "Opt-in to first-party data collection" should remove said cookie.
* The status of your data preference should be reflected in the in-page above the buttons.
